Sources/VSM/StateContainer/StateContainer.swift

```diff
@@ -63,6 +58,20 @@ final public class StateContainer<State>: ObservableObject, StateContaining {
     deinit {
         cancelRunningObservations()
     }
+    
+    // MARK: - StatePublishing
+    
+    /// Publishes the state on `didSet` (main thread). For a `willSet` publisher, use the `$state` projected value.
+    @available(*, deprecated, renamed: "didSetPublisher", message: "It has been renamed to didSetPublisher and will be removed in a future version.")
```

Comment:
nit - just say "Renamed to didSetPublisher and will be removed in a future version."
"It has been" is ambiguous and passive tense.